storage/src/main/java/org/apache/kafka/storage/internals/log/RemoteStorageThreadPool.java

```diff
@@ -45,9 +48,13 @@ public RemoteStorageThreadPool(String threadNamePattern,
                 ThreadUtils.createThreadFactory(threadNamePattern, false,
                         (t, e) -> LOGGER.error("Uncaught exception in thread '{}':", t.getName(), e))
         );
-        metricsGroup.newGauge(REMOTE_LOG_READER_TASK_QUEUE_SIZE_METRIC.getName(),
+        deprecatedLogMetricsGroup.newGauge(REMOTE_LOG_READER_TASK_QUEUE_SIZE_METRIC.getName(),
```

Comment:
Could you please use `DEPRECATE_REMOTE_LOG_READER_TASK_QUEUE_SIZE_METRIC` for consistency?